CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 
 ## master (unreleased)
 
+* Remove "minitest/autorun" and "minitest/pride" requirement from "rubocop/minitest/support". ([@bquorning][])
```

Comment:
Can you create an entry file instead of directly editing the CHANGELOG.md?

Reply:
Yes, sorry. It’s done now.